.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
pytest? I think this file should not be here ;-)

Reply:
@bolkedebruin Oops, you're right. Accidentally added the file by mistake. I've removed the file.